ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/influxdb.j2

```diff
@@ -0,0 +1,57 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - influxdb #}
+{% if influxdb is arista.avd.defined %}
+!
+monitor telemetry influx
+{%     if influxdb.destinations is arista.avd.defined %}
+{%         for destination in influxdb.destinations %}
+{%             if destination.name is arista.avd.defined %}
+   destination influxdb {{ destination.name }}
+{%             endif %}
+{%             if destination.url is arista.avd.defined %}
+      url {{ destination.url }}
+{%             endif %}
+{%             if destination.database is arista.avd.defined %}
+      database name {{ destination.database }}
+{%             endif %}
+{%             if destination.policy is arista.avd.defined %}
+      retention policy {{ destination.policy }}
+{%             endif %}
+{%             if destination.vrf is arista.avd.defined %}
+      vrf {{ destination.vrf }}
```

Comment:
Since vrf is remaining constant for all destinations keep it common for all. (Check for source as well).

Reply:
Created vrf key under influxdb and removed it from destination.